apps/client/src/services/generated/core/CancelablePromise.ts

```diff
@@ -0,0 +1,127 @@
+/* istanbul ignore file */
```

Comment:
this is some spicy codegen - what's the process to generate? Is this something we need to include as part of `npm build`?